pkg/dashboard/resource/function.go

```diff
@@ -158,6 +170,12 @@ func (fr *functionResource) Update(request *http.Request, id string) (attributes
 		return
 	}
 
+	err = fr.validateUpdateInfo(functionInfo, functions[0])
```

Comment:
The problem here is this:
In the imported state, after the function has the `skip-build` and `skip-deploy` annotations removed, if the user tries to disable the function, it will in turn build and deploy the function and then disable it.
That is why @omesser and I thought to return a BadRequest if someone tries to disable an imported function before it is deployed.

Reply:
This is an important note then, can you add that as a comment?

Reply:
Definitely